build.sbt

```diff
@@ -0,0 +1,35 @@
+import Dependencies._
+import sbt.Keys.fork
+
+ThisBuild / scalaVersion := "2.13.2"
+ThisBuild / version := "0.1.0-SNAPSHOT"
```

Comment:
Is the SNAPSHOT build correct?

Reply:
No, probably not. If this was a library we were releasing then yeah but it's not so I've changed it to 0.1.0